build/android-openh264.sh

```diff
@@ -75,6 +65,10 @@ PREFIX="${BASEDIR}/prebuilt/android-$(get_target_build)/${LIB_NAME}" \
 NDKLEVEL="${API}" \
 NDKROOT="${ANDROID_NDK_ROOT}" \
 NDK_TOOLCHAIN_VERSION=clang \
+TOOLCHAINPREFIX=dummy_prefix \
```

Comment:
Do we need to use these two TOOLCHAIN options? Having them doesn't have an effect for me on Mac.

Reply:
good catch! Updated this. Actually, I also made half a step further, as you can see.

Reply:
Yes, the new version is pretty clean. Thanks